lib/fizzy/instructions.hpp

```diff
@@ -39,4 +41,32 @@ struct InstructionMetrics
 
 const InstructionMetrics* get_instruction_metrics_table() noexcept;
 
+template <typename T, size_t MaxSize>
+struct constexpr_vector
```

Comment:
Why have this in `instructions.hpp` and not in `utils.hpp` or something like that? Also should have specific tests for it.

Reply:
I put it into separate file, now I'll make another PR for it.

Reply:
https://github.com/wasmx/fizzy/pull/415